### PR TITLE
indicate the changes of HTTPChaos behaviors

### PR DESCRIPTION
**Pull request by Hexilee**
Signed-off-by: xixi <i@hexilee.me>

Complete #261, please review the [rendered document](https://github.com/Hexilee/website/blob/update-http-chaos/docs/simulate-http-chaos-on-kubernetes.md#specify-side).

### CODE REVIEW

File: docs/simulate-http-chaos-on-kubernetes.md [hunk 1]
```diff
@@ -159,9 +160,86 @@ The `Response` is a meaningful when the `target` set to `Response` during the fa
...
+  port: 80
+  path: "*"
+  request_headers:
+    Host: "nginx.nginx.svc"
+  abort: true
+```
+
+In other cases, especially when injecting the inbound request from outside, you may select the request/response by the `X-Forwarded-Host` header in the request.
+
+For example, if you want to interrupt all requests to you server behind a public gateway `nginx.host.org`, you can apply the following YAML config:
```
Comment (Oreoxmt) [inline docs/simulate-http-chaos-on-kubernetes.md:223]:
```suggestion
For example, if you want to interrupt all requests to your server behind a public gateway `nginx.host.org`, you can apply the following YAML config:
```

File: docs/simulate-http-chaos-on-kubernetes.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ For the detailed description of HTTPChaos configuration, see [Field description]
 Before injecting the faults supported by HTTPChaos, note the followings:
 
 - There is no control manager of Chaos Mesh running on the target Pod.
+- The rules will affect both of clients and servers in the pod, if you want to affect only one side, please refer to the [specify side](#specify-side) section.
```
Comment (Oreoxmt) [inline docs/simulate-http-chaos-on-kubernetes.md:25]:
```suggestion
- The rules will affect both of clients and servers in the Pod, if you want to affect only one side, please refer to the [specify side](#specify-side) section.
```

File: docs/simulate-http-chaos-on-kubernetes.md [hunk 3]
```diff
@@ -159,9 +160,86 @@ The `Response` is a meaningful when the `target` set to `Response` during the fa
...
 | --- | --- | --- | --- | --- | --- |
 | `code` | int32 | Specifies the status code responded by `target`. | Takes effect for all status codes by default. | no | 200 |
-| `response_heads` | map[string]string | Matches request headers to `target`. | Takes effect for all responses by default. | no | Content-Type: application/json |
+| `response_headers` | map[string]string | Matches request headers to `target`. | Takes effect for all responses by default. | no | Content-Type: application/json |
 | `replace.code` | int32 | Specifies the replaced content of the response status code. |  | no | 404 |
 
+
+## Specify side
+
+The rules will affect both of clients and servers in the pod by default, but you can affect only one side by selecting the request headers.
```
Comment (Oreoxmt) [inline docs/simulate-http-chaos-on-kubernetes.md:169]:
```suggestion
The rules will affect both of clients and servers in the Pod by default, but you can affect only one side by selecting the request headers.
```

File: docs/simulate-http-chaos-on-kubernetes.md [hunk 4]
```diff
@@ -159,9 +160,86 @@ The `Response` is a meaningful when the `target` set to `Response` during the fa
...
+
+This section provides some examples to specify the affected side, you can adjust the header selector in rules depend on your particular cases.
+
+### Client side
+
+To inject faults into clients in the pod without affecting servers, you can select the request/response by the `Host` header in request.
```
Comment (Oreoxmt) [inline docs/simulate-http-chaos-on-kubernetes.md:175]:
```suggestion
To inject faults into clients in the Pod without affecting servers, you can select the request/response by the `Host` header in the request.
```

File: docs/simulate-http-chaos-on-kubernetes.md [hunk 5]
```diff
@@ -159,9 +160,86 @@ The `Response` is a meaningful when the `target` set to `Response` during the fa
...
+  port: 80
+  path: "*"
+  request_headers:
+    Host: "example.com"
+  abort: true
+```
+
+### Server side
+
+To inject faults into servers in the pod without affecting clients, you can also select the request/response by the `Host` header in the request.
```
Comment (Oreoxmt) [inline docs/simulate-http-chaos-on-kubernetes.md:199]:
```suggestion
To inject faults into servers in the Pod without affecting clients, you can also select the request/response by the `Host` header in the request.
```

File: docs/simulate-http-chaos-on-kubernetes.md [hunk 6]
```diff
@@ -159,9 +160,86 @@ The `Response` is a meaningful when the `target` set to `Response` during the fa
...
+
+For example, if you want to interrupt all requests to you server behind service `nginx.nginx`, you can apply the following YAML config:
```
Comment (Oreoxmt) [inline docs/simulate-http-chaos-on-kubernetes.md:201]:
```suggestion
For example, if you want to interrupt all requests to your server behind service `nginx.nginx.svc`, you can apply the following YAML config:
```

File: docs/simulate-http-chaos-on-kubernetes.md (hunk 1 above)
Comment (Hexilee) [reply docs/simulate-http-chaos-on-kubernetes.md]: Fixed

File: docs/simulate-http-chaos-on-kubernetes.md (hunk 2 above)
Comment (Hexilee) [reply docs/simulate-http-chaos-on-kubernetes.md]: Fixed

File: docs/simulate-http-chaos-on-kubernetes.md (hunk 3 above)
Comment (Hexilee) [reply docs/simulate-http-chaos-on-kubernetes.md]: Fixed

File: docs/simulate-http-chaos-on-kubernetes.md (hunk 4 above)
Comment (Hexilee) [reply docs/simulate-http-chaos-on-kubernetes.md]: Fixed

File: docs/simulate-http-chaos-on-kubernetes.md (hunk 5 above)
Comment (Hexilee) [reply docs/simulate-http-chaos-on-kubernetes.md]: Fixed

File: docs/simulate-http-chaos-on-kubernetes.md (hunk 6 above)
Comment (Hexilee) [reply docs/simulate-http-chaos-on-kubernetes.md]: Fixed

**Review by Oreoxmt — APPROVED** [review]

**Review by Andrewmatilde — APPROVED** [review]
LGTM

File: docs/simulate-http-chaos-on-kubernetes.md [hunk 7]
```diff
@@ -159,9 +160,86 @@ The `Response` is a meaningful when the `target` set to `Response` during the fa
...
 | `replace.code` | int32 | Specifies the replaced content of the response status code. |  | no | 404 |
 
+
+## Specify side
+
+The rules will affect both of clients and servers in the Pod by default, but you can affect only one side by selecting the request headers.
+
+This section provides some examples to specify the affected side, you can adjust the header selector in rules depend on your particular cases.
+
+### Client side
```
Comment (cwen0) [inline docs/simulate-http-chaos-on-kubernetes.md:173]: From the description, the configuration difference between `client side` and `server side` is mainly the difference in selector?

Comment (Hexilee) [reply docs/simulate-http-chaos-on-kubernetes.md:173]: Yes, this is a rough workaround in the current version. We will have a more precise option to distinguish them.

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**